bin/autofilter.py

```diff
@@ -7,7 +7,7 @@
 import argparse
 import textwrap
 
-VERSION = "V1.0.2"
+VERSION = "V1.0.3"
```

Comment:
This would be 1.1.0 as adding sourmash is a new feature not a fix or minor update